h/views/activity.py

```diff
@@ -96,6 +97,7 @@ class GroupSearchController(SearchController):
     def __init__(self, group, request):
         super(GroupSearchController, self).__init__(request)
         self.group = group
+        self._org_resource = OrganizationResource(group.organization, request)
```

Comment:
It looks like the traversal for this view uses the `h.models.group.GroupFactory` which returns a `group` model, not a `GroupResource` (with an `OrganizationResource` being created directly).

Because of that, alas, `self.group.organization.logo !== self._org_resource.logo` because `self.group` is a _model_, not a _resource_ and as such does not have the `logo` URL built (`self.group.organization.logo` would be the SVG source of the logo).

I believe what should happen here _ultimately_ is that the view should take as its context a `GroupResource`, not a `group` model (as @seanh is alluding to; doable by changing the `h.routes` configuration for this view) but that that is probably beyond the scope of what can be done is short order, presuming we want to get this merged in a timely fashion. Which means that what's happening here—the direct instantiation of an Organization Resource—may unfortunately be necessary for the short term.

However, it is my opinion, orthogonally, that this view is in near-critical need of refactor—it's nearly 550 lines long, for Pete's sake! It's grown organically complex and I worry about its ability to continue to evolve without some love and care. And I think that if we decide to merge this PR the way it is in this respect, we should absolutely plan to come back and update this view's context to use a `GroupResource` instead of a model.